services/src/main/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionService.java

```diff
@@ -74,6 +78,8 @@
 public class TargetApplicationSubmissionService extends AbstractSdtService implements
         ITargetApplicationSubmissionService {
 
+    private static final String CLAIM_NUMBER = "claimNumber";
```

Comment:
This constant doesn't seem to be used in this class.  Is it needed?

Reply:
removed